src/components/PeopleManagement/GroupCardGrid.jsx

```diff
@@ -33,6 +33,7 @@ const GroupCardGrid = ({ groups }) => {
         <Collapsible
           styling="basic"
           title={`Show all ${groups.length} groups`}
+          className="group-collapsible"
```

Comment:
nit: `.groups-collapsible`? (rationale: seems like the title implies multiple groups?